client/tailscale/apitype/apitype.go

```diff
@@ -32,3 +32,9 @@ type WaitingFile struct {
 	Name string
 	Size int64
 }
+
+// SetDeviceToken is the body POSTed to /set-device-token.
```

Comment:
SetDeviceTokenRequest ?

mention LocalAPI in the docs too, as this package is a weird mix.